src/backend/src/app.js

```diff
@@ -24,6 +25,13 @@ const MAPBOX_ACCESS_TOKEN =
   'pk.eyJ1IjoiZGlnaWRlbSIsImEiOiJjbHRyaGh3cm0wN3l4Mmpsam95NDI3c2xiIn0.daq2iZFZXQ08BD0VZWAGUw'
 const DEFAULT_ONLINE_MAP_STYLE_URL = `https://api.mapbox.com/styles/v1/mapbox/outdoors-v11?access_token=${MAPBOX_ACCESS_TOKEN}`
 
+// needed to have logs captured by sentry
+debug.log = function log(...args) {
+  // @ts-expect-error InspectOpts shouldn't be undefined but might be
+  return console.log(formatWithOptions(debug.inspectOpts, ...args))
```

Comment:
This change fixes the undefined issue. We still need the @ts-expect-error, because the way [debug parses env variables](https://github.com/debug-js/debug/blob/33330fa8616b9b33f29f7674747be77266878ba6/src/node.js#L124-L145) means that the opts could have incorrect values, but I think it's ok to skip validating that since we are not setting them from env at this time.

```suggestion
  return console.log(formatWithOptions(debug.inspectOpts || {}, ...args))
```